deploy/helm/scf/assets/operations/instance_groups/diego-cell.yaml

```diff
@@ -42,30 +41,88 @@
-      #         command: ['sh', '-c', 'ss -nlt | grep "LISTEN.*:17019"']
-      #     liveness: *probe
+  path: /instance_groups/name=diego-cell/jobs/name=rep/properties/set_kernel_parameters?
+  value: false
 
 # Temporarily remove garden network_plugin.
 - type: remove
   path: /instance_groups/name=diego-cell/jobs/name=garden/properties/garden/network_plugin
 - type: remove
   path: /instance_groups/name=diego-cell/jobs/name=garden/properties/garden/network_plugin_extra_args
 
+# Add bosh_containerization properties for the rep.
+- type: replace
+  path: /instance_groups/name=diego-cell/jobs/name=rep/properties/bosh_containerization?
+  value:
+    ports:
+    - name: rep
+      protocol: TCP
+      internal: 1800
+    - name: rep-tls
+      protocol: TCP
+      internal: 1801
+
 - type: replace
   path: /instance_groups/name=diego-cell/jobs/name=cflinuxfs3-rootfs-setup/properties/quarks?/bpm/processes
   value: []
 - type: replace
   path: /instance_groups/name=diego-cell/jobs/name=cfdot/properties/quarks?/bpm/processes
   value: []
+
+- type: replace
+  path: /instance_groups/name=diego-cell/jobs/name=rep/properties?/diego?/rep?/locket?/api_location?
+  value: ((deployment-name))-diego-api:8891
+
+- type: replace
+  path: /instance_groups/name=diego-cell/jobs/name=rep/properties?/diego?/rep?/bbs?/api_location?
+  value: ((deployment-name))-diego-api:8889
+
+# TODO: the namespace here has to be determined automatically - helm can help here?
+- type: replace
+  path: /instance_groups/name=diego-cell/jobs/name=rep/properties?/diego?/rep?/advertise_domain?
+  value: scf.svc.cluster.local
+
+# TODO: this should actually be the container's hostname
+# - type: replace
+#   path: /instance_groups/name=diego-cell/jobs/name=rep/properties?/diego?/rep?/cell_id?
+#   value: scf-dev-diego-cell-v1-0
+
+- type: replace
+  path: /instance_groups/name=diego-cell/jobs/name=rep/properties?/diego?/rep?/preloaded_rootfses?
+  value:
+  - cflinuxfs3:/var/vcap/data/rep/cflinuxfs3/rootfs.tar
+
+- type: replace
+  path: /instance_groups/name=diego-cell/jobs/name=garden/properties?/garden?/  _pool?
+  value: 10.38.0.0/16
+
+- type: replace
+  path: /instance_groups/name=diego-cell/jobs/name=garden/properties?/garden?/apparmor_profile?
+  value: ""
+
+- type: replace
+  path: /instance_groups/name=diego-cell/jobs/name=garden/properties?/garden?/containerd_mode?
+  value: false
+
+
+- type: replace
+  path: /instance_groups/name=diego-cell/jobs/name=garden/properties?/garden?/cleanup_process_dirs_on_wait?
+  value: false
+
+- type: replace
+  path: /instance_groups/name=diego-cell/jobs/name=garden/properties?/garden?/disable_swap_limit?
+  value: true
+
+- type: replace
+  path: /instance_groups/name=diego-cell/jobs/name=rep/properties?/bosh_containerization?/privileged?
+  value: true
+
+- type: replace
+  path: /variables/name=diego_rep_agent_v2/options/alternative_names?
+  value:
+  - ((deployment-name))-diego-cell
+  - "*.((deployment-name))-diego-cell"
```

Comment:
This won't work with the default cluster CoreDNS. Please leave it commented out with a TODO like we do for other alternative_names in other certificates.